cmd/gentypes/gentypes.go

```diff
@@ -173,6 +173,19 @@ func emitToplevelType(typeName string, descJson json.RawMessage) string {
 		log.Fatal(err)
 	}
 
+	var comment string
+	descDescriptionJson, ok := descMap["description"]
```

Comment:
descriptionJson would be more consistent with line 166 and easier to read.

Reply:
Done